chef/cookbooks/cpe_screensaver/README.md

```diff
@@ -24,3 +28,18 @@ For example, you could tweak the above values
 
     node.default['cpe_screensaver']['idleTime'] = 300
     node.default['cpe_screensaver']['askForPasswordDelay'] = 2
+    node.node['cpe_screensaver']['MESSAGE'] = 'Unauthorised access is prohibited.'
+
+    # Acceptable (defaults to KenBurns) values: Floating, Flipup, Reflections, Origami,
+    # ShiftingTiles, SlidingPanels, PhotoMobile, HolidayMobile, PhotoWall, VintagePrints,
+    # KenBurns, Classic OR a custom path '/Users/YOURUSERNAME/Pictures'
+    node.default['cpe_screensaver']['styleKey']
+
+    # Acceptable (defaults to 1-National Geographic) values: 1-National Geographic, 2-Aerial, 3-Cosmos, 4-Nature Patterns
+    node.default['cpe_screensaver']['SelectedFolderPath'] = 1-National Geographic
+
+    # Acceptable (defaults to 0) values: 0  or 1
+    node.default['cpe_screensaver']['ShufflesPhotos'] = 0
+
+Although you can manage the screensaver "module" via a profile, the UI doesn't get locked down. This could be misleading to the end user, they may think they can change it via the UI and will find out that their settings keep getting mashed by the profile. Consider a lockdown profile "DisabledPreferencePanes" to work in conjuntion with this cookbook.
```

Comment:
80 cols
do you have rubocop and foodcritic installed ? Please run them against your changes